applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/app_ZDebugMCUBoot.conf

```diff
@@ -0,0 +1,153 @@
+#
```

Comment:
Let's also have only one configuration with MCUBoot added by default.

Reply:
Yes, that's how it's done now. I remembered. Let me finish tests and I will force push my change :)